bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/InvalidDataException.java

```diff
@@ -0,0 +1,17 @@
+package org.openhab.binding.myq.internal;
+
+import java.io.IOException;
+
+/**
+ * Throw if the data we are parsing in not what we are expecting for input. 
+ * @author Dan Cunningham
+ *
+ */
+public class InvalidDataException extends IOException {
```

Comment:
Is `InvalidDataException` conceptually a subclass of `IOException`?  Since the transport guarantees that delivered data is not corrupted due to network problems, it seems to me that invalid data could not be the result of an I/O problem.